src/pickers/HealthFacilityReferPicker.js

```diff
@@ -0,0 +1,68 @@
+import { healthFacilityLabel, LOCATION_SUMMARY_PROJECTION } from "../utils";
```

Comment:
Once you removed location from query, this projection is not used anymore. Please, remove it.